libs/dh/admin/feature-user-management/src/lib/invite/dh-invite-user-modal.component.ts

```diff
@@ -214,34 +216,35 @@ export class DhInviteUserModalComponent extends WattTypedModal {
     const [prefix, ...rest] = phoneParts;
     const formattedPhoneNumber = `${prefix} ${rest.join('')}`;
 
-    this.inviteUserStore.inviteUser({
-      invitation: {
-        invitationUserDetails:
-          firstname.value && lastname.value && phoneNumber.value
-            ? {
-                firstName: firstname.value,
-                lastName: lastname.value,
-                phoneNumber: formattedPhoneNumber,
-              }
-            : undefined,
-        email: email.value,
-        assignedActor: actorId.value,
-        assignedRoles: this.userRoles.controls.selectedUserRoles.value,
-      },
-      onSuccess: () => this.onInviteSuccess(email.value),
-      onError: (e) => this.onInviteError(e),
-    });
+    this.inviteUserMutation
+      .mutate({
+        variables: {
+          input: {
+            userInviteDto: {
+              invitationUserDetails:
+                firstname.value && lastname.value && phoneNumber.value
+                  ? {
+                      firstName: firstname.value,
+                      lastName: lastname.value,
+                      phoneNumber: formattedPhoneNumber,
+                    }
+                  : undefined,
+              email: email.value,
+              assignedActor: actorId.value,
+              assignedRoles: this.userRoles.controls.selectedUserRoles.value,
+            },
+          },
+        },
+      })
+      .then(() => this.onInviteSuccess(email.value))
+      .catch((e) => this.onInviteError(e));
```

Comment:
question: it's been long time since I last saw a Promise :D Why not using `onCompleted`, `onError` options?

Reply:
I forgot to change it :-) nice catch